다각형의 넓이 #1

sejungkim opened the pull request:
다각형의 넓이 첫번째 버전입니다.

sejungkim committed:
Complete calculateCircle function

sejungkim committed:
Complete calculateSquare function

sejungkim committed:
Complete calculateTrapezoid function

crongro commented on index.js:
```diff
@@ -1,50 +1,33 @@
-        size = radius1 * radius1 * Math.PI;
-        console.log("원의 넓이: " + size);
+function checkNumberOfInput(validNum, inputVal) {
+    if(inputVal.length !== validNum) {
```
짧은 내용임으로 3항연산자를 써보세요.

crongro commented on index.js:
```diff
@@ -1,50 +1,33 @@
-        console.log("사각형의 넓이: " + size);
+function calculateCircle(radius) {
+    if(!checkNumberOfInput(1, arguments)) {
+        return console.log("1개의 인자가 필요합니다.");
```
console.log 결과를 return 하는건가요?  return이 필요할까요?

sejungkim replied:
함수의 종료를 명시적으로 보여주려고 그랬던건데, 없는게 나을까요?

crongro commented on index.js:
```diff
@@ -1,50 +1,33 @@
-    if(arguments.length === 1){
-        size = radius1 * radius1 * Math.PI;
-        console.log("원의 넓이: " + size);
+function checkNumberOfInput(validNum, inputVal) {
```
validNum 아이디어는 좋네요.

sejungkim replied:
감사합니다!

crongro reviewed (changes requested):
함수구분을 잘 했어요. 
리뷰반영해보세요.

sejungkim committed:
Apply review

- apply conditional ternary operator
- remove return in all calculate functions

sejungkim commented:
리뷰 반영 했습니다.

crongro reviewed (approved):
프로그램이 실행되다가 오류가 생기면, 프로그램을 멈추고 오류를 뱉어야 하는 경우가 생길 수 있어요.
지금은 오류가 아니죠.

그럴때는 
 throw "숫자가 필요해요";  이렇게도 합니다.

지금 구현한것과 차이점을 확인해보세요.
